lib/puppet/pops/time/timestamp.rb

```diff
@@ -33,10 +33,13 @@ def self.parse(str, format = DEFAULT_FORMATS)
     end
   end
 
-  undef_method :-@, :+@, :%,:modulo,:divmod, :div, :fdiv, :abs, :abs2, :magnitude # does not make sense on a Timestamp
+  undef_method :-@, :+@, :div, :fdiv, :abs, :abs2, :magnitude # does not make sense on a Timestamp
   if method_defined?(:negative?)
     undef_method :negative?, :positive?
   end
+  if method_defined?(:%)
+    undef_method :%, :modulo, :divmod
```

Comment:
whether `modulo` and `divmod` are defined doesn't seem tied to `%`. My jruby Numeric has both of them but not `%`.